content/rom_manual/epd/questionnaires.md

```diff
@@ -70,6 +70,21 @@ Selecteer vervolgens de gewenste lijsten en klik op "Invullen". Je gaat meteen d
 
 ## Speciale gevallen
 
+### Direct invullen van een vragenlijst
+
+Soms is het wenselijk om een vragenlijst direct in te kunnen vullen. Dit kan door een vragenlijst te selecteren en op de knop Direct invullen te klikken. Er verschijnt nu een scherm waarin u de gewenste situatie kunt kiezen.
+
+<img src="/assets/images/screenshots/dossier_quests11.png" />
+
+Kies de situatie die van toepassing is. "Interview" wanneer u samen met de patiënt een vragenlijst wilt invullen. "Overnemen van papier" is vooral handig voor bulk-invoer. Klik op "Verder" om te beginnen met invullen.
```

Comment:
"is vooral handig voor bulk-invoer" gooit de deur een beetje open voor ander gebruik, terwijl de optienaam zelf zo lekker duidelijk is.